node/libs/crypto/src/bls12_381/mod.rs

```diff
@@ -21,18 +22,18 @@ pub const INFINITY_PUBLIC_KEY: [u8; PUBLIC_KEY_BYTES_LEN] = [
     0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
 ];
 
+const KEY_GEN_SALT: &[u8] = b"ERA-CONSENSUS-BLS12381-SIG-KEYGEN-SALT";
```

Comment:
https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-bls-signature-05#name-choosing-a-salt-value-for-k
says that RECOMMENDED choice of salt is a uniformly random sequence of bytes

Reply:
It will be a uniformly random sequence of bytes after hashing, which is performed

Reply:
That's not how salt works. The whole point of HKDF is to AVOID assuming that hash function produces output of good enough entropy: https://www.rfc-editor.org/rfc/rfc5869.txt . It also says that salt is useful iff key material is not guaranteed to be uniformly random. In our case we always use strong entropy source, so we don't need salt at all.

Reply:
I'm just saying to stick to rfc, rather that come up with your own crypto scheme, which is what I'd expect you to suggest in the first place.

Reply:
also the naming is inconsistent with the rfc: "salt_hash" should be just "salt" and KEY_GEN_SALT should be KEY_GEN_SALT_PREIMAGE or sth similar.

Reply:
ok